SU-SOE/stanford_soe_helper
# 7.x 2.x soe 2891 callout

# READY FOR REVIEW

# Summary
This adds the `highlight` (formerly `callout`) styles. After the WYSIWYG has been configured, it will allow you to apply the highlight styles to a paragraph.

To avoid merge conflicts, this was branched off of and depends on the branch [SOE-2763-2-column-css](https://github.com/SU-SOE/stanford_soe_helper/tree/SOE-2763-2-column-css). 

# Needed By (5/4/2018)
- Sprint end

# Criticality
- Fixed on prod, need to put into code.

# Steps to Test
- Switch to this branch `7.x-2.x-SOE-2891-callout`

- Create a new magazine article
- Add a simple paragraph
- Create paragraph of text
- Switch *Text format* to: *Full-html*
- Add to the paragraph: `class="highlight"`
- Save 
- Verify it looks like this:

![screen shot 2018-05-02 at 13 25 12](https://user-images.githubusercontent.com/284440/39547390-4b5db56c-4e0c-11e8-9f04-9a04aea294c9.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2891

## Related PRs
To avoid merge conflicts this PR is dependent on 
https://github.com/SU-SOE/stanford_soe_helper/pull/168

## More Information

## Folks to notify
@kerri-augenstein 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
